bindings/pyfiction/test/inout/test_write_sidb_layout_to_svg_in_siqad_layout.py

```diff
@@ -0,0 +1,206 @@
+import unittest
+from mnt.pyfiction import *
+
+def normalize_svg(svg: str) -> str:
+    """
+    Normalizes an SVG string by removing all whitespace, including spaces, tabs, and newlines.
+
+    This function takes an SVG string as input and returns a new string with all
+    whitespace characters removed. This can be useful for comparing SVG strings
+    or reducing the size for transmission.
+
+    Parameters:
+    svg (str): The input SVG string to be normalized.
+
+    Returns:
+    str: A string with all whitespace characters removed.
+    """
+    # Remove all whitespace (spaces, tabs, newlines)
+    return ''.join(svg.split())
+
+cell_level_dark_mode = """<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generated by fiction v0.6.5 (https://github.com/cda-tum/fiction) -->
```

Comment:
These strings hardcode the fiction version and repo. These should be set dynamically or these tests will fail on the next update. See the existing C++ tests for file writing. Maybe we have to expose global variables like  `__version__` and `__repo__` in Python first, though. It's easy enough since C++ already sets them dynamically. We can simply bind those to Python variables.